sdk/test-utils/recorder/package.json

```diff
@@ -99,6 +98,46 @@
     "rimraf": "^3.0.0",
     "tslib": "^2.2.0",
     "ts-node": "^10.0.0",
```

Comment:
Should be able to remove `ts-node` with the server moved to `.mjs`.  Also should be able to upgrade `tslib` too.

Reply:
Updated in a later commit, just couldn't push, pre-push blocked me.